code/modules/mob/living/silicon/robot/robot_modules.dm

```diff
@@ -318,154 +309,115 @@
 	emag_modules = list(/obj/item/borg/stun)
 	ratvar_modules = list(
 		/obj/item/clockwork/slab/cyborg/engineer,
-		/obj/item/clockwork/replica_fabricator/cyborg)
+		/obj/item/clockwork/clockwork_proselytizer/cyborg)
 	cyborg_base_icon = "engineer"
 	moduleselect_icon = "engineer"
 	feedback_key = "cyborg_engineering"
 	magpulsing = TRUE
 	hat_offset = INFINITY // No hats
 
-/obj/item/robot_module/security
-	name = "Security"
-	basic_modules = list(
-		/obj/item/device/assembly/flash/cyborg,
-		/obj/item/restraints/handcuffs/cable/zipties/cyborg,
-		/obj/item/melee/baton/loaded,
-		/obj/item/gun/energy/disabler/cyborg,
-		/obj/item/clothing/mask/gas/sechailer/cyborg)
-	emag_modules = list(/obj/item/gun/energy/laser/cyborg)
-	ratvar_modules = list(/obj/item/clockwork/slab/cyborg/security,
-		/obj/item/clockwork/weapon/ratvarian_spear)
-	cyborg_base_icon = "sec"
-	moduleselect_icon = "security"
-	feedback_key = "cyborg_security"
-	can_be_pushed = FALSE
-	hat_offset = 3
-
-/obj/item/robot_module/security/do_transform_animation()
-	..()
-	to_chat(loc, "<span class='userdanger'>While you have picked the security module, you still have to follow your laws, NOT Space Law. \
-	For Asimov, this means you must follow criminals' orders unless there is a law 1 reason not to.</span>")
-
-/obj/item/robot_module/security/respawn_consumable(mob/living/silicon/robot/R, coeff = 1)
-	..()
-	var/obj/item/gun/energy/e_gun/advtaser/cyborg/T = locate(/obj/item/gun/energy/e_gun/advtaser/cyborg) in basic_modules
-	if(T)
-		if(T.cell.charge < T.cell.maxcharge)
-			var/obj/item/ammo_casing/energy/S = T.ammo_type[T.select]
-			T.cell.give(S.e_cost * coeff)
-			T.update_icon()
-		else
-			T.charge_tick = 0
-
-/obj/item/robot_module/peacekeeper
+/obj/item/weapon/robot_module/peacekeeper
 	name = "Peacekeeper"
 	basic_modules = list(
 		/obj/item/device/assembly/flash/cyborg,
-		/obj/item/cookiesynth,
+		/obj/item/weapon/cookiesynth,
 		/obj/item/device/harmalarm,
-		/obj/item/reagent_containers/borghypo/peace,
-		/obj/item/holosign_creator/cyborg,
+		/obj/item/weapon/reagent_containers/borghypo/peace,
+		/obj/item/weapon/holosign_creator/cyborg,
 		/obj/item/borg/cyborghug/peacekeeper,
-		/obj/item/extinguisher,
-		/obj/item/borg/projectile_dampen)

```

Comment:
Let's remove one of the few things that makes the PK any good and make it so that people only ever pick engineering module since it's the only thing that'll be viable after all the removals